sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -674,18 +675,19 @@
                 },
                 "supportsHttpsTrafficOnly": true,
                 "allowBlobPublicAccess": false,
-                "networkAcls": "[if(equals(parameters('storageAccountBehindVNet'), 'true'), variables('networkRuleSetBehindVNet'), json('null'))]",
+                "networkAcls": "[if(and(equals(parameters('storageAccountBehindVNet'), 'true'), equals(parameters('set_up_feature_store'), 'false')), variables('networkRuleSetBehindVNet'), json('null'))]",
```

Comment:
should offline store container also config networkAcls?

Reply:
it's always false as before

Reply:
removed set up feature store condition